### ISSUE TITLE
Firebase Issues with Speechly

### ISSUE THREAD

**Issue by thoughtcastapp** [speechly/speechly#251]
### Describe the bug
When you have a project with Firebase, you get a error: missing required module 'CNIOAtomics'

### To Reproduce
I have attached a sample project with the latest build of Firebase. If you notice, the speechly libraries are not present in the main app (but firebase is). The app clip has speechy libraries.

[SpeechlyTest.zip](https://github.com/speechly/speechly/files/10191909/SpeechlyTest.zip)

### Expected behaviour
The Main app (SpeechlyTest) should build and it doesn't.

### Screenshots
<img width="497" alt="image" src="https://user-images.githubusercontent.com/38260217/206628190-49dbabda-8ef6-4a68-84ca-1d76b8caa8b0.png">

### Environment

- Platform: Mobile
- OS: iOS
- Browser: n/a
- Version: Xcode 14.1
- Package version: Firebase 10.3.0, Speechly 0.8.9

**Comment by thoughtcastapp** [comment]
Worth mentioning, I think I found a solution to this...the library for speechly was not being added to my main target. Once put there, everything seemed to work...I don't know why this isn't being done automatically but this is a usable solution:

![image](https://user-images.githubusercontent.com/38260217/206760490-e97223f1-43f4-4359-b607-0d9fc5a9643e.png)

**Comment by langma** [comment]
I agree, `Speechly` should be in the target list. I'm not sure if this can be achieved via the package declaration, but I'll investigate and report back here.
